src/bindings/js/node/src/tensor.cpp

```diff
@@ -191,3 +192,49 @@ Napi::Value TensorWrap::is_continuous(const Napi::CallbackInfo& info) {
     }
     return Napi::Boolean::New(env, _tensor.is_continuous());
 }
+
+Napi::Value TensorWrap::set_shape(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+
+    if (info.Length() != 1 || !info[0].IsArray()) {
```

Comment:
Try to validate it using `ov::js::validate` as [here](https://github.com/openvinotoolkit/openvino/blob/300b9304f30391d14ca83d03a59b2ff5315956ac/src/bindings/js/node/src/model_wrap.cpp#L181).